### DISCUSSION TITLE
Suppress AM/PM and use 24 hour clock

### DISCUSSION THREAD

**Discussion by Blue-Moses** [Extended-Thunder/send-later#364] [General]
How do I use a 24 hour clock in SEND LATER? I find AM/PM confusing and wish to use only 24 hour clock to avoid mistakes.

**Comment by jph-sendlater** [comment]
This feature will be available in version 9.0. Currently, the date/time formats are controlled by some combination of your operating system settings, and your Thunderbird locale preferences. Send Later 9.0 will include a preference for overriding those system settings (at least in the way dates are formatted -- there isn't much I can do about the calendar date/time picker widget in the scheduler dialog)

**Comment by jikamens** [comment]
The current version of Send Later does a better job of using the same date/time format as your OS, so if you have the OS configured to use a 24-hour clock then Send Later should as well. Independent of that, you can now customize the date formats that Send Later users, as documented [in the user guide](https://extended-thunder.github.io/send-later/#customize-datetime-formats).
